programs/rtcweb.c

```diff
@@ -120,10 +120,10 @@ struct peer_connection {
 #define SCTP_PACKED
 #endif
 
-#if defined(_WIN32)
+#if defined(_WIN32) && !defined(__MINGW32__)
 #pragma warning( push )
 #pragma warning( disable : 4200 )
-#endif //defined(_WIN32)
```

Comment:
While you are there: Can you change it to a C-style comment?

Reply:
Done!